admyral/actions/integrations/compliance/github.py

```diff
@@ -0,0 +1,103 @@
+"""
+Setup:
+Follow setup instruction from the github documentation to create personal access token
+https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+"""
+
+from typing import Annotated
+from httpx import Client
+
+from admyral.action import action, ArgumentMetadata
+from admyral.context import ctx
+from admyral.typings import JsonValue
+
+
+def get_github_enterprise_client(access_token: str, enterprise: str) -> Client:
+    return Client(
+        base_url=f"https://api.github.com/enterprises/{enterprise}",
+        headers={
+            "Authorization": f"Bearer {access_token}",
+            "Accept": "application/vnd.github.v3+json",
+        },
+    )
+
+
+@action(
+    display_name="Query Organizaton Audit Logs",
```

Comment:
```suggestion
    display_name="Search Enterprise Audit Logs",
```

Search for consistency reasons.